Server/src/routes/playerRoute.ts

```diff
@@ -70,10 +71,10 @@ playerRoute.get(
     let year = req.params['year'];
     let currentVersion = await getCurrentVersion(year);
     let playerIdRange = getTeamIdRange(year);
-    let query = `SELECT * FROM players WHERE playerId > ${playerIdRange.lowerBound} AND playerId < ${playerIdRange.upperBound} AND version > ${version}`;
```

Comment:
we should get rid of the select * in future

Reply:
created a jira: [BP-86](https://rskennedy.atlassian.net/browse/BP-86)